openflow-sdn/openflow-protocol.md

```diff
@@ -0,0 +1,65 @@
+# What is the OpenFlow protocol? 
+
+**Openflow protocol** is a standard in Software Defined Networking(SDN) architecture. This protocol defines the communication between an SDN controller and the network device/agent. Before delving deep into the OpenFlow protocol, here is a background on SDN. 
+
+Before moving further, the term "switch" denotes a network device capable of using OF protocol, and not the Layer 2 device. 
+
```

Comment:
Please add a sentence or two linking the switch, layer 2 device, and the OSI layer. A beginner might wonder what you mean by a switch not being a "Layer 2 device".